openbb_platform/providers/fmp/tests/test_fmp_fetchers.py

```diff
@@ -762,3 +763,24 @@ def test_fmp_historical_market_cap_fetcher(credentials=test_credentials):
     fetcher = FmpHistoricalMarketCapFetcher()
     result = fetcher.test(params, credentials)
     assert result is None
+
+
+@pytest.mark.record_http
+def test_fmp_government_trades_fetcher(credentials=test_credentials):
+    """Test FMP government trades fetcher.
+    params limit only functions when there is no parameter symbol
+    """
+    params = {
+        "chamber": "all",
```

Comment:
Here, we will be capturing the requests. If we can minimize the amount of data received in the HTTP request, this will help reduce the overall footprint of the testing suite. A limit of 1 should be adequate as the purpose for these tests is to check the fetching mechanism and the typing.

Capturing the test cassettes is rather peculiar, so don't worry about that. I'll deal with that, you've done great.